docker/rx-mission/app.py

```diff
@@ -0,0 +1,19 @@
+import json
+
+from flask import Flask, jsonify
+
+app = Flask(__name__)
+
+
+@app.route("/product/<string:product_id>", methods=["GET"])
+def handle_request(product_id):
+    with open("./fixtures/rx-mission-products.json", encoding="utf-8") as json_file:
+        parsed_json = json.load(json_file)
```

Comment:
possibly we can do this in the `if __name__` block so that it's performed once at startup time instead of for every request?

Reply:
I've moved the reading and processing of the json file to the `if __name__` block